react-native/react/local-debug.desktop.js

```diff
@@ -18,8 +18,8 @@ let config = {
   reduxDevToolsEnable: false,
   redirectOnLogout: true,
   reduxDevToolsSelect: state => state, // only watch a subset of the store
-  enableStoreLogging: false,
-  forwardLogs: false
+  enableStoreLogging: true,
+  forwardLogs: true
```

Comment:
We have to be careful as this logging can be slow. Console. Log is
synchronous

On Wednesday, January 20, 2016, Marco Munizaga <notifications@github.com>
wrote:

> In react-native/react/local-debug.desktop.js
> <https://github.com/keybase/client/pull/1783#discussion_r50351975>:
>
> > @@ -18,8 +18,8 @@ let config = {
> >    reduxDevToolsEnable: false,
> >    redirectOnLogout: true,
> >    reduxDevToolsSelect: state => state, // only watch a subset of the store
> > -  enableStoreLogging: false,
> > -  forwardLogs: false
> > +  enableStoreLogging: true,
> > +  forwardLogs: true
>
> It makes sense to enable this stuff otherwise the log is somewhat useless.
>
> Also with store logging we can even re-enter the same state the user was
> in!
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/keybase/client/pull/1783/files#r50351975>.
>